tests/Unit/Helpers/Twitter/Image_Helper_Test.php

```diff
@@ -50,4 +59,41 @@ public function test_get_image_size() {
 
 		$this->assertEquals( 'full', $this->instance->get_image_size() );
 	}
+
+	/**
+	 * Tests retrieval of an image by its id.
+	 *
+	 * @covers ::get_by_id
+	 * @dataProvider get_by_id_data_provider
+	 *
+	 * @param bool   $is_valid Whether the attachment is valid.
+	 * @param string $expected The expected result.
+	 * @param int    $times    The number of times get_attachment_image_source should be called.
+	 *
+	 * @return void
+	 */
+	public function test_get_by_id( $is_valid, $expected, $times ) {
+		$this->image
+			->expects( 'is_valid_attachment' )
+			->with( 1337 )
+			->once()
+			->andReturn( $is_valid );
+
+		$this->image
+			->expects( 'get_attachment_image_source' )
+			->times( $times )
+			->andReturn( $expected );
+
+		$this->assertEquals( $expected, $this->instance->get_by_id( 1337 ) );
+	}
+
+	/**
+	 * Data provider for the test_get_by_id test.
+	 *
+	 * @return array<bool,string,int> The data for the test.
```

Comment:
Reading https://psalm.dev/docs/annotating_code/type_syntax/array_types/
Comma (`,`) is separating key, value. While the pipe (`|`) is separating multiple type possibilities.
```suggestion
	 * @return array<string,bool|string|int> The data for the test.
```